src/bindings/html/dom.js

```diff
@@ -49,17 +49,17 @@ function getTranslatables(element) {
     nodes, element.querySelectorAll('*[data-l10n-id]'));
 }
 
-export function translateDocument(doc, langs) {
-  let setDOMLocalized = function() {
-    doc.localized = true;
-    dispatchEvent(doc, 'DOMLocalized', langs);
-  };
+export function setDOMLocalized(langs) {
+  this.localized = true;
+  dispatchEvent(this, 'DOMLocalized', langs);
```

Comment:
In other exported functions in this file, `this` is an instance of the `View`.  Would it be more consistent to keep it that way here and pass  `doc` as the first argument to `setDOMLocalized`?